src/bartlett.jl

```diff
@@ -45,12 +45,12 @@ function BartlettTest(X::AbstractMatrix, Y::AbstractMatrix)
     return BartlettTest(L′, p, nx, ny)
 end
 
-StatsBase.nobs(B::BartlettTest) = (B.nx, B.ny)
-StatsBase.dof(B::BartlettTest) = div(B.p * (B.p + 1), 2)
+StatsAPI.nobs(B::BartlettTest) = (B.nx, B.ny)
```

Comment:
If we're defining methods for these for some tests, we should reexport them

Reply:
I exported them in https://github.com/JuliaStats/HypothesisTests.jl/pull/297/commits/d83c2fd2da5b29676c7a4d7c984d0f044f2f860a. Thereby I could also address your comment in https://github.com/JuliaStats/HypothesisTests.jl/pull/297#discussion_r1187655952: The imports are removed completely.

Reply:
![image](https://github.com/JuliaStats/HypothesisTests.jl/assets/6396159/1e1878d5-69d5-4182-b61a-68dda01b1bfc)